### PR TITLE
Completed task-8

### PR DESCRIPTION
**Pull request by Martin-Abbott**

### CODE REVIEW

**Review by HannahBurks — COMMENTED** [review]
If I click post multiple times to post a comment, the page goes black and crashes. Look how you can prevent multiple clicks and stop the button from being clicked whilst the post goes through.

File: MA-FE-NC-GAMES/components/CommentAdder.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			.catch((err) => {
+				setErr(
+					`${err.response.data.msg}. Please refresh the page and try again.`
+				);
+			});
+	};
+
+	return (
+		<>
+			<form onSubmit={handleSubmit}>
```
Comment (HannahBurks) [inline MA-FE-NC-GAMES/components/CommentAdder.jsx:47]: We want to add some form validation here to prevent us being able to post an empty comment :)

**Review by HannahBurks — COMMENTED** [review]
If I click post multiple times to post a comment, the page goes black and crashes. Look how you can prevent multiple clicks and stop the button from being clicked whilst the post goes through.

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Nice start Martin but some form related errors to be handled. We also need to ensure we don't post multiple of the same comment . Once implemented please send another PR tomorrow :) nice work today!

**Review by aloush-dev — APPROVED** [review]
nice job merge away

**Pull request merged**